extensions/chromium/contentscript.js

```diff
@@ -132,8 +132,11 @@ function updateEmbedElement(elem) {
   }
   elem.type = "text/html";
   elem.src = getEmbeddedViewerURL(elem.src);
-  if (parentNode) {
+
+  if (nextSibling) {
     nextSibling.before(elem);
+  } else if (parentNode) {
+    parentNode.append(elem);
```

Comment:
```suggestion
  if (parentNode) {
    // Suppress linter warning: insertBefore is preferable to
    // nextSibling.before(elem) because nextSibling may be null.
    // eslint-disable-next-line prefer-modern-dom-apis
    parentNode.insertBefore(elem, nextSibling);
  }
```

Here is an alternative that fixes the issue without triggering the linter.

Reply:
fixed https://github.com/mozilla/pdf.js/pull/16612/commits/dce076e44b77f859757522aaa582a5211ac5101c

Reply:
Please see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

Reply:
@Snuffleupagus Thanks.

fix commits with squash and tweak commit message. https://github.com/mozilla/pdf.js/pull/16612/commits/fcf38f6ae9623a01bd67d0dff8f99042255f993f